variables.tf

```diff
@@ -0,0 +1,68 @@
+variable "acm_certificate" {
+  type    = object({
+    arn = string
+  })
+  default = null
+
+  description = <<EOS
+The ARN of the ACM certificate that you want to use with the CloudFront distribution.
+If not specified, the default CloudFront certificate for *.cloudfront.net will be used.
+
+This only makes sense in combination with `aliases`.
+EOS
+}
+
+variable "acm_certificate_minimum_protocol_version" {
+  type    = string
+  default = "TLSv1"
```

Comment:
my proposal was to use `TLSv1.3` as default here, as every new instance of this module should start with an up-to-date TLS version, but downgrade the TLS version on the existing module instances (and tackle the update in a separate story).